src/index.js

```diff
@@ -150,16 +223,35 @@ const isHidden = function (node, displayCheck) {
   if (matches.call(nodeUnderDetails, 'details:not([open]) *')) {
     return true;
   }
+  const displayCheck = options.displayCheck;
+  const getShadowRoot = options.getShadowRoot || noop;
```

Comment:
Could be written with more modern destructuring like:
```javascript
const {
  displayCheck,
  getShadowRoot = noop
} = options;
```

Reply:
👍